lib/Document/Adapter/Ghostscript.php

```diff
@@ -180,7 +180,7 @@ public function saveImage(string $imageTargetPath, int $page = 1, int $resolutio
             Console::addLowProcessPriority($cmd);
             $process = new Process($cmd);
             $process->setTimeout(240);
-            $process->run();
+            $process->mustRun();
```

Comment:
If you are using `mustRun` vs `run`, you probably want to catch `ProcessFailedException` and then log that as well in `getCacheFileStream`:

>The mustRun() method is identical to run(), except that it will throw a [ProcessFailedException](https://github.com/symfony/symfony/blob/7.2/src/Symfony/Component/Process/Exception/ProcessFailedException.php) if the process couldn't be executed successfully (i.e. the process exited with a non-zero code):

Reply:
ignore this